tests/test_client.py

```diff
@@ -66,6 +67,9 @@ def test_generic_get_methods(rester):
     )
 
     if name not in ignore_generic:
+        warnings.filterwarnings(
+            "ignore", "get_data_by_id is deprecated", DeprecationWarning
```

Comment:
Need confirm on this, I'm not sure if this test is intended to test deprecated method `get_data_by_id` or should we replace it?
```
tests/test_client.py::test_generic_get_methods[PhononRester]
tests/test_client.py::test_generic_get_methods[ProvenanceRester]
tests/test_client.py::test_generic_get_methods[RobocrysRester]
tests/test_client.py::test_generic_get_methods[SimilarityRester]
  /Users/yang/developer/api/tests/test_client.py:82: DeprecationWarning: get_data_by_id is deprecated and will be removed soon. Please use the search method instead.
    doc = rester.get_data_by_id(
```

Reply:
We probably don't need to test the deprecated method but I'd leave the warnings in for now.